### ISSUE TITLE
TextEditor.get is deprecated.

### ISSUE THREAD

**Issue by vasaki** [bsegraves/custom-folds#27]
`TextEditor.prototype.displayBuffer` has always been private, but now
it is gone. Reading the `displayBuffer` property now returns a reference
to the containing `TextEditor`, which now provides *some* of the API of
the defunct `DisplayBuffer` class.
```
TextEditor.get (C:\Users\Vasily Vlasyuk\AppData\Local\atom\app-1.15.0\resources\app.asar\src\text-editor.js:105:14)
<unknown> (C:\Users\Vasily Vlasyuk\.atom\packages\custom-folds\lib\index.js:217:21)
Function.module.exports.Emitter.simpleDispatch (C:\Users\Vasily Vlasyuk\AppData\Local\atom\app-1.15.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
Emitter.emit (C:\Users\Vasily Vlasyuk\AppData\Local\atom\app-1.15.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:129:28)
<unknown> (C:\Users\Vasily Vlasyuk\AppData\Local\atom\app-1.15.0\resources\app.asar\src\workspace.js:265:34)
Function.module.exports.Emitter.simpleDispatch (C:\Users\Vasily Vlasyuk\AppData\Local\atom\app-1.15.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
```

**Comment by 0tho** [comment]
this issue is the same as #25

**Comment by bsegraves** [comment]
Fixed in 1.9.4.

**Comment by vasaki** [comment]
Thank you!!
